10up/brightcove-video-connect
# Update node version and dependencies

### Description of the Change


This PR updates node version to a minimum of 16.


Closes  https://github.com/10up/brightcove-video-connect/issues/328

### How to test the Change


```
* Run npm install
* Run npm build
* Assets should build correctly
```

### Changelog Entry

> Updated - Node 16 and dependencies


### Credits

Props @oscarssanchez , @felipeelia , ...


### Checklist:
